OpenRA.Mods.Common/UtilityCommands/LegacyMapImporter.cs

```diff
@@ -180,32 +178,38 @@ public void ConvertIniMap(string iniFile)
 			LoadActors(file, "INFANTRY");
 			LoadSmudges(file, "SMUDGE");
 
-			foreach (var p in players)
-				LoadPlayer(file, p, legacyMapFormat == IniMapFormat.RedAlert);
-
 			var wps = file.GetSection("Waypoints")
 					.Where(kv => Exts.ParseIntegerInvariant(kv.Value) > 0)
 					.Select(kv => Pair.New(Exts.ParseIntegerInvariant(kv.Key),
 						LocationFromMapOffset(Exts.ParseIntegerInvariant(kv.Value), mapSize)));
-
 			// Add waypoint actors
```

Comment:
[SA1515] A single-line comment must be preceded by a blank line or another single-line comment, or must be the first item in its scope. To ignore this error when commenting out a line of code, begin the comment with '////' rather than '//'.